### PR TITLE
Update CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by RahulSharma099**

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -21,10 +21,11 @@ Examples of unacceptable behavior by participants include:
 * Public or private harassment
 * Publishing others' private information, such as a physical or electronic address, without explicit permission
 * Other conduct which could reasonably be considered inappropriate in a professional setting
+* Use of unacceptable commit text or username is also prohibited.
 
 ## Our Responsibilities
 
-Project maintainers are responsible for clarifying the standards of acceptable behavior and are expected to take appropriate and fair corrective action in response to any instances of unacceptable behavior.
+Project maintainers are responsible for clarifying the standards of acceptable behavior and are expected to take appropriate and fair corrective action in response to any instance of unacceptable behavior.
```
Comment (waellerbe) [inline CODE_OF_CONDUCT.md]: @RahulSharma099  I applaud your attention to detail for the typo found in the "Code of Conduct" Markdown file.  Changed noted along Line #28 in the revised file.

📓  Learning how to communicate in the proper manner on GitHub.

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -21,10 +21,11 @@ Examples of unacceptable behavior by participants include:
 * Public or private harassment
 * Publishing others' private information, such as a physical or electronic address, without explicit permission
 * Other conduct which could reasonably be considered inappropriate in a professional setting
+* Use of unacceptable commit text or username is also prohibited.
```
Comment (waellerbe) [inline CODE_OF_CONDUCT.md]: Line #24 : "Use of unacceptable commit text or username is also prohibited."

As someone who is **new** to the experience of commenting on a GitHub project, can someone clarify what this rule means? I am hoping that I am not confusing **username** with the @mention feature on GitHub. Thank you in return for helping my understanding of this matter.

**Comment by coliff** [comment]
The Code of Conduct is adapted from this repo: https://github.com/ContributorCovenant/contributor_covenant

I'd personally suggest proposing any changes upstream to that repo. I see that there is a [v2 on the horizon](https://github.com/ContributorCovenant/contributor_covenant/pull/709)...

**Comment by mdo** [comment]
Agreed with @coliff's comment. Benefit of upstream changes is everyone can benefit from it as that project is updated :). Please open an issue here if y'all catch the new release before us.

**Pull request closed without merging**